sbncode/CAFMaker/CMakeLists.txt

```diff
@@ -41,6 +41,8 @@ art_make_library( LIBRARY_NAME sbncode_CAFMaker
                   systematicstools::interface
                   systematicstools::interpreters
                   systematicstools::utility
+                  ${GENIE_LIB_LIST}
```

Comment:
Is this needed? where is this defined?
In the best case, this is actually empty and can be removed.

Reply:
This was required for
```
#include "Framework/EventGen/EventRecord.h"
#include "Framework/Ntuple/NtpMCEventRecord.h"
```
and further usage of these objects in making GENIE tree. These are defined from [nufinder](https://github.com/NuSoftHEP/nufinder/blob/develop/Modules/FindGENIE.cmake#L44-L70).